app/helpers/settings_users_helper.rb

```diff
@@ -0,0 +1,106 @@
+module SettingsUsersHelper
+  private
+
+  def settings_users_summary(record)
+    summary = [
+      settings_users_basic_info(record),
+      settings_users_groups_info(record)
+    ]
+    safe_join(summary)
+  end
+
+  def settings_users_basic_info(record)
+    rows = [
+      row_data(_('ID'), record.id),
+      row_data(_('Full Name'), record.name),
+      row_data(_('Username'), record.userid),
+      row_data(_('E-mail Address'), record.email),
+    ]
+    if record.current_group
+      row = {
+        :cells => {
+          :label => _('Current Group'),
+          :value => record.current_group.description,
+          :icon  => "ff ff-group",
+        }
+      }
+      if role_allows?(:feature => "rbac_group_show")
+        row[:cells][:link] = record.current_group.description
+        row[:cells][:onclick] = "miqOnClickSelectRbacTreeNode('g-#{record.current_group.id}')"
+        row[:cells][:title] = _("View this Group")
+      else
+        row[:cells][:link] = "#"
+      end
+      rows.push(row)
+    else
+      rows.push(row_data(_('Current Group'), ''))
+    end
+
+    if record.current_group&.miq_user_role
+      row = {
```

Comment:
Lines 20 to 34 are again replicated here, can you check the feasibility of putting this in a function and passing the values to it and getting `row` as the return object?

Reply:
@akhilkr128 made the changes . please check